.circleci/config.yml

```diff
@@ -18,37 +18,35 @@ checkout step for each job: &addWorkspace
 default config for js: &js_defaults
   <<: *defaults
   docker:
-    - image: circleci/node:14
+    - image: cimg/node:14.18.1
 
 default config for macOS: &macos_defaults
   <<: *defaults
   resource_class: 'medium'
   macos:
-    xcode: '13.2.1'
+    xcode: '13.1.0'
```

Comment:
Why are we downgrading Xcode here?

Reply:
Yeah, I went one minor down, as 13.1.0 has node `14.18.1`, while 13.2.1 has `16.13.1`, to keep version consistent across executors